cmd/token.go

```diff
@@ -0,0 +1,192 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/compute/metadata"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/go-tpm-tools/client"
+	"github.com/google/go-tpm-tools/launcher/agent"
+	"github.com/google/go-tpm-tools/launcher/spec"
+	"github.com/google/go-tpm-tools/launcher/verifier"
+	"github.com/google/go-tpm-tools/launcher/verifier/rest"
+	"github.com/google/go-tpm/legacy/tpm2"
+	"github.com/spf13/cobra"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+var mdsClient *metadata.Client
+
+// If hardware technology needs a variable length teenonce then please modify the flags description
+var tokenCmd = &cobra.Command{
+	Use:   "token",
+	Short: "Attest and fetch an OIDC token from Google Attestation Verification Service.",
+	Long: `Gather attestation report and send it to Google Attestation Verification Service for an OIDC token.
+The OIDC token includes claims regarding the GCE VM, which is verified by Attestation Verification Service. Note that Confidential Computing API needs to be enabled for your account to access Google Attestation Verification Service https://pantheon.corp.google.com/apis/api/confidentialcomputing.googleapis.com.
```

Comment:
I believe the external-facing URL is console.cloud.google.com :).

So, https://console.cloud.google.com/apis/api/confidentialcomputing.googleapis.com